src/tools/dev/scripts/bv_support/bv_vtkm.sh

```diff
@@ -200,10 +199,13 @@ function build_vtkm
 
     vopts=""
     vopts="${vopts} -DCMAKE_INSTALL_PREFIX:PATH=${VISITDIR}/vtkm/${VTKM_VERSION}/${VISITARCH}"
-    vopts="${vopts} -DVTKm_ENABLE_TESTING:BOOL=OFF"
+    vopts="${vopts} -DVTKm_ENABLE_TESTING:BOOL=ON"
     vopts="${vopts} -DVTKm_ENABLE_RENDERING:BOOL=ON"
-    vopts="${vopts} -DVTKm_USE_64BIT_IDS:BOOL=OFF"
+    vopts="${vopts} -DVTKm_USE_64BIT_IDS:BOOL=ON"
     vopts="${vopts} -DVTKm_USE_DOUBLE_PRECISION:BOOL=ON"
+    vopts="${vopts} -DVTKm_USE_DEFAULT_TYPES_FOR_VTK:BOOL=ON"
+    vopts="${vopts} -DVTKm_ENABLE_DIY_NOMPI:BOOL=ON"
+    vopts="${vopts} -DBUILD_SHARED_LIBS:BOOL=OFF"
     vopts="${vopts} -DCMAKE_BUILD_TYPE:STRING=${VISIT_BUILD_MODE}"
```

Comment:
Lines 202 through 208 should be replaced with:
```
    vopts="${vopts} -DVTKm_ENABLE_TESTING:BOOL=OFF"
    vopts="${vopts} -DVTKm_ENABLE_TESTING_LIBRARY:BOOL=ON"
    vopts="${vopts} -DVTKm_ENABLE_RENDERING:BOOL=ON"
    vopts="${vopts} -DVTKm_USE_64BIT_IDS:BOOL=OFF"
    vopts="${vopts} -DVTKm_USE_DOUBLE_PRECISION:BOOL=ON"
    vopts="${vopts} -DVTKm_USE_DEFAULT_TYPES_FOR_VTK:BOOL=ON"
    vopts="${vopts} -DCMAKE_BUILD_TYPE:STRING=${VISIT_BUILD_MODE}"
    vopts="${vopts} -DBUILD_SHARED_LIBS:BOOL=OFF"
```
We want testing off, but the testing library on. We also don't want 64 bit ids. We don't want shared libraries.